### PR TITLE
Add Alteration/CancerType models && Update FdaSubmission Model

### PR DESCRIPTION
**Pull request by zhx828**
This fixes https://github.com/oncokb/oncokb/issues/3042 and https://github.com/oncokb/oncokb/issues/3041

### CODE REVIEW

File: jhipster-jdl.jdl [hunk 1]
```diff
@@ -76,34 +76,93 @@ entity CompanionDiagnosticDevice {
     manufacturer String
 }
 
-entity PremarketApproval {
-	device String
-    genericName String
-    applicant TextBlob
+enum FdaSubmissionType {
+	PMA, DE_NOVO, HDE, FIVE_TEN_K
```
Comment (zhx828) [inline jhipster-jdl.jdl:80]: @calvinlu3 is it appropriate to name it FIVE_TEN_K ?

File: jhipster-jdl.jdl [hunk 2]
```diff
@@ -76,34 +76,93 @@ entity CompanionDiagnosticDevice {
...
+}
+
+entity FdaSubmission {
```
Comment (zhx828) [inline jhipster-jdl.jdl:83]: We still need to add the specimen type. But not sure whether it should be in FdaSubmission or CompanionDiagnosticDevice. Waiting for confirmation from Ritika.

File: jhipster-jdl.jdl [hunk 3]
```diff
@@ -76,34 +76,93 @@ entity CompanionDiagnosticDevice {
...
+
+enum CopyNumberAlterationType {
+	AMPLIFICATION,
+    DELETION,
+    GAIN,
+    LOSS
+}
+
+enum StructuralVariantType {
+	FUSION
```
Comment (zhx828) [inline jhipster-jdl.jdl:113]: Temporarily just add FUSION for now.

File: jhipster-jdl.jdl (hunk 1 above)
Comment (calvinlu3) [reply jhipster-jdl.jdl:80]: The FDA also uses `PMN` (Premarket Notification).
https://www.accessdata.fda.gov/scripts/cdrh/cfdocs/cfpmn/pmn.cfm?ID=K173492

Comment (zhx828) [reply jhipster-jdl.jdl:80]: Yeah I saw that too. But the abbreviation seems only be used in the URL?

**Pull request merged**